### DISCUSSION TITLE
Multitenancy Fresh installation /admin/1 gives 404 Error

### DISCUSSION THREAD

**Discussion by khanakia** [filamentphp/filament#9188] [Help]
I did install and set up the multinenancy in multiple projects in the past and worked fine.

Today i tried to set in some new Laravel projects with Fresh installation https://filamentphp.com/docs/3.x/panels/tenancy#setting-up-tenancy

When I go to http://localhost:7001/admin/1 it gives 404 error

Note: i have followed and doubled checked all the steps and also compared the code with other projects.

Any idea what it could be?

**Comment by khanakia** [comment]
Just fixed: Forgot to implement the the classes `class User extends Authenticatable implements FilamentUser, HasDefaultTenant, HasTenants`

**Reply by maytham553** [reply]
thanks , you save my day
